obal/data/modules/nevr.py

```diff
@@ -0,0 +1,38 @@
+"""
+Calculate NEVRA for a package
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.obal import get_specfile_nevr, get_specfile_name
+
+
+def main():
+    """
+    Calculate name-epoch-version-release for a specfile
+    """
+    module = AnsibleModule(
+        argument_spec=dict(
+            spec_file=dict(type='str', required=True),
+            scl=dict(type='str', required=False, default=None),
+            dist=dict(type='str', required=False, default=None),
```

Comment:
required=False will result in `None` in the case of no input, no need to set `default`